Tests/DependencyInjection/SonataPageExtensionTest.php

```diff
@@ -0,0 +1,92 @@
+            'Sonata\\PageBundle\\CmsManager\\CmsManagerSelectorInterface',
+            'Sonata\\PageBundle\\CmsManager\\CmsPageManager',
+            'Sonata\\PageBundle\\CmsManager\\CmsSnapshotManager',
+            'Sonata\\PageBundle\\CmsManager\\DecoratorStrategy',
+            'Sonata\\PageBundle\\CmsManager\\DecoratorStrategyInterface',
+            'Sonata\\PageBundle\\Entity\\BaseBlock',
+            'Sonata\\PageBundle\\Entity\\BasePage',
+            'Sonata\\PageBundle\\Entity\\BaseSite',
+            'Sonata\\PageBundle\\Entity\\BaseSnapshot',
+            'Sonata\\PageBundle\\Entity\\BlockInteractor',
+            'Sonata\\PageBundle\\Entity\\BlockManager',
+            'Sonata\\PageBundle\\Entity\\PageManager',
+            'Sonata\\PageBundle\\Entity\\SiteManager',
+            'Sonata\\PageBundle\\Entity\\SnapshotManager',
+            'Sonata\\PageBundle\\Entity\\Transformer',
+            'Sonata\\PageBundle\\Generator\\Mustache',
+            'Sonata\\PageBundle\\Listener\\ExceptionListener',
+            'Sonata\\PageBundle\\Listener\\RequestListener',
+            'Sonata\\PageBundle\\Listener\\ResponseListener',
+            'Sonata\\PageBundle\\Model\\Block',
+            'Sonata\\PageBundle\\Model\\BlockInteractorInterface',
+            'Sonata\\PageBundle\\Model\\Page',
+            'Sonata\\PageBundle\\Model\\PageBlockInterface',
+            'Sonata\\PageBundle\\Model\\PageInterface',
+            'Sonata\\PageBundle\\Model\\PageManagerInterface',
+            'Sonata\\PageBundle\\Model\\Site',
+            'Sonata\\PageBundle\\Model\\SiteInterface',
+            'Sonata\\PageBundle\\Model\\SiteManagerInterface',
+            'Sonata\\PageBundle\\Model\\Snapshot',
+            'Sonata\\PageBundle\\Model\\SnapshotChildrenCollection',
+            'Sonata\\PageBundle\\Model\\SnapshotInterface',
+            'Sonata\\PageBundle\\Model\\SnapshotManagerInterface',
+            'Sonata\\PageBundle\\Model\\SnapshotPageProxy',
+            'Sonata\\PageBundle\\Model\\Template',
+            'Sonata\\PageBundle\\Page\\PageServiceManager',
+            'Sonata\\PageBundle\\Page\\PageServiceManagerInterface',
+            'Sonata\\PageBundle\\Page\\Service\\BasePageService',
+            'Sonata\\PageBundle\\Page\\Service\\DefaultPageService',
+            'Sonata\\PageBundle\\Page\\Service\\PageServiceInterface',
+            'Sonata\\PageBundle\\Page\\TemplateManager',
+            'Sonata\\PageBundle\\Page\\TemplateManagerInterface',
+            'Sonata\\PageBundle\\Request\\SiteRequestContext',
+            'Sonata\\PageBundle\\Route\\CmsPageRouter',
+            'Sonata\\PageBundle\\Site\\BaseSiteSelector',
+            'Sonata\\PageBundle\\Site\\HostPathSiteSelector',
+            'Sonata\\PageBundle\\Site\\HostSiteSelector',
+            'Sonata\\PageBundle\\Site\\SiteSelectorInterface',
+            'Sonata\\PageBundle\\Twig\\Extension\\PageExtension',
+            'Sonata\\PageBundle\\Twig\\GlobalVariables',
```

Comment:
To clarify, if you have this method 

```php
public function getName()
{
   return 'toto';
}
```

It's probably better to test that it returns a string, rather than testing it returns `toto`, do you see why?